profile/README.md

```diff
@@ -26,7 +26,7 @@
 
 <sub>Node.js is an open-source project, and it's always looking for new contributions. From documentation, translation, contributing to our infrastructure or reporting a bug; Any contribution is valued and welcome. Are you interested in contributing to Node.js? Give a read to our [Governance Model](https://github.com/nodejs/node/blob/main/GOVERNANCE.md) and the numerous ways you can [Get Involved](https://nodejs.org/en/get-involved) with Node.js!</sub>
 
-#### 🦺 Help us making this Community safe.
+#### 🦺 Making this Community safe.
```

Comment:
I personally prefer this proposal, but all three LGTM
```suggestion
#### 🦺 Help us make this Community safe.
```

Reply:
@aduh95 Thank you! I've implemented that option here